app_utils/sections/experiment.py

```diff
@@ -1617,22 +1618,34 @@ async def experiment_download_model(q: Q, error: str = ""):
         model.backbone.save_pretrained(checkpoint_path)
         tokenizer.save_pretrained(checkpoint_path)
 
+        card = get_model_card(cfg, model, repo_id="<path_to_local_folder>")
+        card.save(os.path.join(experiment_path, "model_card.md"))
+
         zf = zipfile.ZipFile(zip_path, "w")
 
         FILES_TO_PUSH = [
-            "pytorch_model.bin",
             "vocab.json",
+            "sentencepiece.bpe.model",
+            "bpe_encoder.bin",
             "tokenizer_config.json",
             "tokenizer.json",
             "special_tokens_map.json",
             "merges.txt",
             "generation_config.json",
             "config.json",
             "added_tokens.json",
+            "model_card.md",
         ]
 
         for file in FILES_TO_PUSH:
-            add_file_to_zip(zf=zf, path=f"{experiment_path}/{file}")
+            path = os.path.join(experiment_path, file)
+            if os.path.isfile(path):
+                add_file_to_zip(zf=zf, path=path)
+
+        # Add model weight files
+        weight_files = glob.glob(os.path.join(checkpoint_path, "pytorch_model*.bin"))
```

Comment:
there is also a `pytorch_model.bin.index.json`.
Should we just change the glob to `pytorch_model*.*` ?

Reply:
Good catch, fixed it